site/src/Librecores/ProjectRepoBundle/RepoCrawler/GitRepoCrawler.php

```diff
@@ -88,6 +98,14 @@ class GitRepoCrawler extends RepoCrawler
 
     private $repoClonePath = null;
 
+    protected $projectMetricsProvider;
+
+    public function __construct(SourceRepo $repo, MarkupToHtmlConverter $markupConverter, ProcessCreator $processCreator, ObjectManager $manager, LoggerInterface $logger, ProjectMetricsProvider $projectMetricsProvider)
```

Comment:
- Style: Please split this line into two (at least).
- What makes the project metrics git-specific to be included only in the GitRepoCrawler, and not in the more general RepoCrawler? It looks to me that it should go there.